src/Components/AnsibleTabs/AnsibleTabs.js

```diff
@@ -121,123 +138,122 @@ InstallAnsibleTab.propTypes = {
-        <div className='ins-c-install-client-tab__links'>
-            <Text component={TextVariants.a} onClick={() => setActiveTab(0)}>
-                {intl.formatMessage(messages.roleVariables)}
-            </Text>
-            <Text component={TextVariants.a} onClick={() => setActiveTab(1)}>
-                {intl.formatMessage(messages.factsInstalled)}
-            </Text>
-            <Text component={TextVariants.a} onClick={() => setActiveTab(2)}>
-                {intl.formatMessage(messages.examples)}
-            </Text>
-        </div>
-        {activeTab === 0 && <React.Fragment>
-            <Text component={TextVariants.h4}>
-                {intl.formatMessage(messages.rolesAndConfig)}
-            </Text>
-            <Text> {intl.formatMessage(messages.rolesAndConfigDescription)} </Text>
-            {getSections(intl).map(section => <div className='ins-c-install-client-tab__section' key={section.name}>
-                <ExpandableSection toggleText={section.name}>
-                    {section.description}
-                </ExpandableSection>
-                <hr/>
-            </div>)}
-        </React.Fragment>}
-        {activeTab === 1 && <React.Fragment>
-            <Text component={TextVariants.h4}>
-                {intl.formatMessage(messages.factsInstalled)}
-            </Text>
-            <Text> {intl.formatMessage(messages.factsInstalledOne)} </Text>
-            <Text> {intl.formatMessage(messages.factsInstalledTwo)} </Text>
-            <Text> {intl.formatMessage(messages.exampleTaskOne)} </Text>
-            <pre>
-                <code>
-                    - debug: var=ansible_local.insights.system_id
-                </code>
-            </pre>
-            <Text> {intl.formatMessage(messages.exampleTaskTwo)} </Text>
-        </React.Fragment>}
-        {activeTab === 2 && <React.Fragment>
-            <Text component={TextVariants.h4}>
-                {intl.formatMessage(messages.examples)}
-            </Text>
-            <Text> <strong>{intl.formatMessage(messages.examplePlaybook)}</strong> </Text>
-            <Text> {intl.formatMessage(messages.examplePlaybookOne)} </Text>
-            <pre>
-                <code>
+    return <Tabs activeKey={activeTabKey} className='ins-c-install-client-tab' isSecondary>
+        <Tab eventKey={0} onClick={() => setActiveTabKey(0)} title={<TabTitleText>{intl.formatMessage(messages.roleVariables)}</TabTitleText>}>
+            <Grid hasGutter>
+                <TextContent>
+                    <Text component={TextVariants.h4}>
+                        {intl.formatMessage(messages.rolesAndConfig)}
+                    </Text>
+                    <Text> {intl.formatMessage(messages.rolesAndConfigDescription)} </Text>
+                </TextContent>
+                <Accordion asDefinitionList>
+                    {getSections(intl).map(section => <AccordionItem key={section.name} className='ins-c-install-client-tab__sections'>
```

Comment:
I like the naming here but let's move it to the `Accordion` tag and update the sass